docs/development.md

```diff
@@ -80,7 +80,12 @@ versions necessary for running [regression tests](#python-tests)
 
 You can also develop DiffKemp directly. For this you need to install the
 necessary [dependencies](installation.md#dependencies) to your system and then
-[build DiffKemp](#build).
+you may [build DiffKemp](#build). Or you can develop DiffKemp without using
+`pip`, as it can be quite slow in iterative development.
+
+For that purpose, if you build DiffKemp with CMake just like in [section about
+build](#build) you can use generated executable located in
+`BUILD_DIR/bin/diffkemp`,
```

Comment:
Maybe I would shortend it
```
The generated executable is located in `BUILD_DIR/bin/diffkemp`
```
or move the sentence to the `build section` because the `build` section should be the same for `nix build` and `local build` and in both cases the binary is located in `BUILD_DIR/bin/diffkemp`.